### ISSUE TITLE
Deleting a message from a Stream no longer appears to be supported

### ISSUE THREAD

**Issue by sfrooster** [nats-io/jetstream#94]
Running both the server and jsm from the latest docker images, the following item from the [readme](https://github.com/nats-io/jetstream/blob/master/README.md#deleting-a-message) complains with the following:

```
f4242d00d098:~# jsm str rmm ORDERS 1 -f
jsm: error: expected command but got "rmm", try --help
```

**Comment by ripienaar** [comment]
Hmm, Indeed somehow the latest docker container isnt running the latest jsm binary - latest binary does support it:

```
# ./jsm stream rmm --help
usage: jsm stream rmm [<flags>] [<stream>] [<id>]

Securely removes an individual message from a Stream
```

I'll make a release today

**Comment by sfrooster** [comment]
I've noticed a few **minor** differences between the documentation and _latest_ (an ack message not appearing when the doc indicates it should, or an info message where the default of "No acknowledgement = false" had been replaced - and thereby improved - with "Acknowledgement = true", or something like that) but I know how difficult it can be to keep documentation up-to-date with a code base - especially one that's moving quickly.  Just a quick note to say that I'm pleasantly surprised by how closely the two seem to track and ultimately only reported this because it was divergent enough.  I won't become one of those "doc hawks".

**Comment by ripienaar** [comment]
Thanks :) I don’t mind doc hawks, especially ones who send PRs :P 

Jokes aside please keep sending comments about inconsistencies. Doc inconsistencies might be because there are bugs and in fact the doc was correct so I am happy to keep reviewing small nit picks

**Comment by sfrooster** [comment]
Then, in that spirit, I'll open one more PR with the small collection of other items I noticed in my brief foray this evening.  To follow...

**Comment by ripienaar** [comment]
Please try again with `synadia/jsm:0.0.8`?

**Comment by sfrooster** [comment]
Yes, this seems to be fixed in `synadia/jsm:0.0.8`
